scripts/generate_signed_upload_tx.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+WASM_FILE=$1
+DEPLOYER=$2
+
+FEES=1000000uosmo
+NODE=https://osmosis-rpc.publicnode.com:443
```

Comment:
If we hardcode it maybe let's use the official Osmosis RPC: `https://rpc.osmosis.zone:443` ?

Reply:
done